test/unit/ready.js

```diff
@@ -67,4 +91,32 @@ QUnit.module( "ready" );
 			"Argument passed to fn in jQuery(document).ready( fn ) should be jQuery" );
 	} );
 
+	QUnit.test( "Promise.resolve(jQuery.ready)", function( assert ) {
+		assert.expect( 2 );
+		var done = jQuery.map( new Array( 2 ), function() { return assert.async(); } );
```

Comment:
I have [updated](https://github.com/jquery/jquery/commit/b97c8d30c5aedace75dc17056d429f28e41b20c1) QUnit to 1.20.0, so after rebase, you can start using new signature in this pull